app/Http/Controllers/Api/APICRUDController.php

```diff
@@ -138,8 +138,11 @@ protected function onUpdate($id, $payload){
     protected function _update($id, array $payload)
     {
         try {
+            if (array_key_exists("groups", $payload)) {
```

Comment:
what is the purpose of this ?
this controller is an abstract one not an specialized one

Reply:
Moved to UserApiController but we need this explicit convertion on server-side bc even though we send an int array the API receives a string array representation (HTTP protocol works in that way)